roles/ciao-common/tasks/main.yml

```diff
@@ -18,8 +18,13 @@
   - name: Create certificate directory
     file: path=/etc/pki/ciao state=directory
 
-  - name: Create /var/lib/ciao
-    file: path=/var/lib/ciao state=directory owner=ciao group=ciao
+  - name: Create /var/lib/ciao tree
```

Comment:
@albertomurillo  how the creation of this is related to  a clean up ?  looks like a miss commit change in this pull